server/application-server/src/main/java/de/tum/in/www1/hephaestus/leaderboard/LeaderboardService.java

```diff
@@ -99,6 +103,10 @@ public List<LeaderboardEntry> createLeaderboard() {
         return leaderboard;
     }
 
+    private boolean isInTimeframe(OffsetDateTime date, Optional<OffsetDateTime> before, OffsetDateTime after) {
```

Comment:
I think the parameter order can be more understandable by having `after` before `before`